walkingastro.lic

```diff
@@ -9,13 +9,14 @@ class WalkingAstro
     settings = get_settings
     @astrology_prediction_skills_survival = settings.astrology_prediction_skills['survival']
     @no_use_scripts = settings.walkingastro_no_use_scripts
+    @tele_storage = settings.telescope_storage
```

Comment:
❌ The `base` and `base-empty` changes need to be in this PR

Reply:
Yup, getting to that. Need to do them one at a time or I'll simply forget, lol. Thank you!